Cargo.lock

```diff
@@ -1,7 +1,5 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 3
-
 [[package]]
```

Comment:
:eyes:

Reply:
Why does this happen?!

Reply:
I honestly don't know and haven't found anything by searching

Reply:
@tgmichel added it accidentally in PR #613 

To be fair, my compiler adds this line pretty often too, I just always had to remember to revert it before.